test/unit/acknowledgement/process-invalid.test.js

```diff
@@ -0,0 +1,300 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+jest.mock('../../../app/config')
+const mockConfig = require('../../../app/config')
+
+jest.mock('../../../app/event')
+const mockEvent = require('../../../app/event')
+
+jest.mock('../../../app/acknowledgement/get-hold-category-name')
+const getHoldCategoryName = require('../../../app/acknowledgement/get-hold-category-name')
+
+jest.mock('../../../app/holds/get-hold-category-id')
+const getHoldCategoryId = require('../../../app/holds/get-hold-category-id')
+
+jest.mock('../../../app/reschedule')
+const holdAndReschedule = require('../../../app/reschedule')
+
+jest.mock('../../../app/reset')
+const { resetPaymentRequestById } = require('../../../app/reset')
+
+const processInvalid = require('../../../app/acknowledgement/process-invalid')
+
+const mockFRN = require('../../mock-frn')
+const { SFI } = require('../../../app/schemes')
+
+let mockAcknowledgement
+let mockAcknowledgementError
+let mockHoldCategoryName
+let mockHoldCategoryId
+
+let schemeId
+let paymentRequestId
+
+describe('send acknowledgement error event', () => {
+  beforeEach(() => {
+    mockConfig.isAlerting = true
+
+    mockAcknowledgement = JSON.parse(JSON.stringify(require('../../mock-acknowledgement')))
+    mockAcknowledgementError = JSON.parse(JSON.stringify(require('../../mock-acknowledgement-error')))
+    mockHoldCategoryName = JSON.parse(JSON.stringify(require('../../../app/constants/hold-categories-names'))).DAX_REJECTION
+    mockHoldCategoryId = 3
+
+    schemeId = SFI
+    paymentRequestId = 1
+
+    mockEvent.sendAcknowledgementErrorEvent.mockReturnValue(undefined)
+
+    getHoldCategoryName.mockReturnValue(mockHoldCategoryName)
+
+    getHoldCategoryId.mockReturnValue(mockHoldCategoryId)
+
+    holdAndReschedule.mockReturnValue({
+      undefined
+    })
+
+    resetPaymentRequestById.mockReturnValue({
+      undefined
+    })
```

Comment:
Let's remove the object notation and move onto 1 line

`holdAndReschedule.mockReturnValue(undefined)`